src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -319,6 +319,10 @@ function RemoteFunctions(experimental) {
             if (this.trigger) {
                 _trigger(element, "highlight", 1);
             }
+            
+            if (!window.event) {
+                element.scrollIntoViewIfNeeded();
```

Comment:
This script is injected in case of MultiBrowserLivePreview, for now the reverse-inspect doesn't work with the multibrowser, I will take up this task once this PR is merged.(I tried a hacky solution to make it work for multibrowser and it was working. I will raise another PR once I am done with proper solution)
Now, I am using window.scrollTo instead of scrollIntoViewIfNeeded.
I didn't use scrollIntoView because it aligns the element either to top or bottom, but I wanted to align it to center, so I have now used window.scrollTo which is supported by all browsers.